DeviceDiagnostics/DeviceDiagnostics.json

```diff
@@ -120,7 +121,7 @@
     },
     "events": {
         "onAVDecoderStatusChanged":{
-            "summary": "Triggered when the most active status of any audio/video decoder/pipeline changes.",
+            "summary": "Triggered when the active status of any audio/video decoder/pipeline changes.",
```

Comment:
Pls change it to "Triggered when the most active status of audio/video decoder/pipeline changes"

Reply:
Modified as per suggestion